jenkins/scripts/integration_test.sh

```diff
@@ -63,16 +63,16 @@ echo "Running in region: $OS_REGION_NAME"
 if [[ "${TESTS_FOR}" == "feature_tests"* || "${TESTS_FOR}" == "e2e_tests"* ]]
 then
     # Four node cluster
-    TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-200GB}"
+    TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-300GB}"
```

Comment:
The thing to check here is that these flavors actually exist. You can do `openstack flavor list` in both Kna1 and Fra1 regions in CityCloud to see if they do. If the flavors don't exist, making a VM with the specs using the GUI with the sliders will create the corresponding flavor in the CLI environment.

Reply:
Yes they are presetn:
Kna1
| d27c5b84-ce03-45bd-a8c8-45094b4df750 | 8C-32GB-300GB              |  32768 |  300 |         0 |     8 | True      |
Fra1
| 3038f8e9-7a6a-4a9c-a6ff-0857f0e0f8c0 | 8C-32GB-300GB    |  32768 |  300 |         0 |     8 | True      |